src/main/java/io/spring/githubchangeloggenerator/github/service/GitHubService.java

```diff
@@ -45,7 +45,7 @@
 
 	private static final Pattern LINK_PATTERN = Pattern.compile("<(.+)>; rel=\"(.+)\"");
 
-	private static final String MILESTONES_URI = "/repos/{owner}/{name}/milestones";
+	private static final String MILESTONES_URI = "/repos/{owner}/{name}/milestones?state=all&sort=due_on&direction=desc&per_page=50";
```

Comment:
I am a bit nervous that sort=due_on would show milestones that don't have a due date first as it is the case on the Web UI. For projects with a large number of milestones, this may be a problem.

Reply:
That doesn't appear to be what happens with the REST API. Testing against spring-projects/spring-boot returns 50 milestones all of which have a non-null `due_on`.

Reply:
Querying just the open milestones confirms that those with a `null` `due_on` appear last when using a descending sort.

Reply:
Ah, the descending sort puts the furthest out milestones first, i.e. it's the same as the Web UI's "Furthest due date". If you switch to an ascending sort so that the most recent milestones appear first then those with no due date appear first as they do in the web UI.